### DISCUSSION TITLE
Bug with OSXPhotos 0.67.7 on Sonoma 14.6.1

### DISCUSSION THREAD

**Discussion by jeremieberrebi** [RhetTbull/osxphotos#1676] [Q&A]
Hello,

I tried to update my Osxphotos folders with the following command as usual
osxphotos export --skip-edited --retry 3  /volumes/LibraryBackup --ramdb --export-by-date --sidecar XMP --report PhotosBackup_{today}.csv --person-keyword --update

and I'm getting this error message
Photos database version: 6000, 9.
Processing persons in photos.
Processing detected faces in photos.
Processing albums.
Something went wrong and osxphotos encountered an error:
no such table: Z_28ASSETS
Writing export database to /volumes/LibraryBackup/.osxphotos_export.db
Crash log written to '/Users/jeremieberrebi/osxphotos_crash.log'
Please file a bug report at https://github.com/RhetTbull/osxphotos/issues with 
the crash log attached.

Crash log attached
[osxphotos_crash.log](https://github.com/user-attachments/files/16861160/osxphotos_crash.log)

Can you help?

**Comment by RhetTbull** [comment]
The current version of OSXPhotos is 0.68.5. Please update to that. Apple changed the schema of the Photos database in 14.6.1 and older versions of OSXPhotos will no longer work.
